lib/src/chat/chat.dart

```diff
@@ -265,9 +264,7 @@ class _ChatState extends State<Chat> with ChatComposer, ChatCreateMixin, InviteM
             actions: <Widget>[
               if (!isGroup)
                 AdaptiveIconButton(
-                  icon: AdaptiveIcon(
-                      icon: IconSource.phone,
-                  ),
+                  icon: AdaptiveIcon(icon: IconSource.phone, ),
```

Comment:
We have, AS applies the default Dart, Flutter rules and probably VSC also does that, this line is a case of "the problem sits in front of the screen". What every I did there, I screwed the formatting manually :/.

As we have multiple IDEs in the project it's hard to apply globally applicable rules. For AS I can export the settings and everyone could import them.
We could also use some command line tooling, but this would lead to additional complexity. I think we can talk about that in the standup tomorrow. :+1: 

In general we need some global formatting, used by everyone, otherwise commits are filled over and over with annoying diffs, so you are absolutely right. :+1:

Reply:
You are totally right here! In general, it should not matter which IDE is used. However, since the settings of these IDEs are as different as the achievable results, what do you think if we as the team simply agreeing on an IDE by convention? I like VSCode very much, but in the end I don't care which IDE I use as long as we use the same settings. Because then it's unnecessary to make comments like mine in the future. 😊